vulnerabilities/package_managers.py

```diff
@@ -97,7 +122,16 @@ async def fetch(self, pkg, session):
         try:
             response = await session.request(method="GET", url=url)
             response = await response.json()
-            versions = set(response["releases"])
+            for version in response["releases"]:
+                if response["releases"][version]:
+                    versions.add(
+                        Version(
+                            value=version,
+                            release_date=parser.parse(
+                                response["releases"][version][-1]["upload_time_iso_8601"]
```

Comment:
Let's use some intermediate variable that makes this clear:
1. we can have several downloads (a wheel, source dist, etc.)
2. we are picking the last one which is entirely arbitrary
3. we are getting the date

Another approach could have been to pick the most recent one or the oldest one.... in all cases this needs to be clarified